TownHallTriviaApp/main/templates/main/gamePlayRoom.html

```diff
@@ -6,70 +6,59 @@
-                <input type="submit"value="Round 6" />
-                <input type="hidden" name="roundId" value="Round_6">
-              </form>
-            </li>
-            <li>
-              <form action="{{ url_for('main.endGame') }}">
-                <input type="submit"value="End Game" />
-              </form>
-            </li>
-          </ul>
-        </nav></strong>
-      </div>
-    </header>
+  <body class="body">
 
-    <div class="teamBanner">
-      <h1 class="teamBannerTitle">TEAM NAME: </h1>
-      <h1 class="teamBannerSeparator"> {{ teamName }}</h1>
-      <h1 class="teamBannerTitle">GAME ID: </h1>
-      <h1> {{ gameId }}</h1>
+    <div class="navMenuBanner">
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 1" />
+        <input type="hidden" name="roundId" value="Round_1">
+      </form>
+
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 2" />
+        <input type="hidden" name="roundId" value="Round_2">
+      </form>
+
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 3" />
+        <input type="hidden" name="roundId" value="Round_3">
+      </form>
+   
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 4" />
+        <input type="hidden" name="roundId" value="Round_4">
+      </form>
+    
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 5" />
+        <input type="hidden" name="roundId" value="Round_5">
+      </form>
+    
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 6" />
+        <input type="hidden" name="roundId" value="Round_6">
+      </form>
+    
+      <form class="navMenu navMenuLastItem" action="{{ url_for('main.endGame') }}">
+        <input type="submit" class="navMenu" value="END GAME" />
+      </form>
     </div>
 
     <div class="container">
-      <div class="titleSmall play">YOU'RE IN</div>
-      <h1 class="title">THE GAME PLAY ROOM</h1><br><br><br>
+      <div class="titleSmall">YOU'RE IN</div>
+      <h1 class="title">THE WAITING ROOM</h1>
+      <br>
+
+    <div class="teamBanner">
+      <h2 class="teamBannerTitle">TEAM NAME: </h2>
+      <h2 class="teamBannerSeparator"> {{ teamName }}</h2>
+      <h2 class="teamBannerTitle">GAME ID: </h2>
+      <h2> {{ gameId }}</h2>
+    </div>
 
+      <br>
       <h2>You're in the game!<br><br>
-        Welcome to the game play room. This will be your homescreen in-between rounds.<br><br>
+        Welcome to the waiting room. This will be your homescreen in-between rounds.<br><br>
```

Comment:
We sure waiting room instead of gamePlay room? Theyre not waiting here, they are playing the game here

Reply:
changed to game lobby